src/Middleware/RateLimiting/src/IRateLimiterMetadata.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// An interface which can be used to identify a type which provides metadata needed for enabling request rate limiting support.
+/// </summary>
+internal interface IRateLimiterMetadata
+{
+    /// <summary>
+    /// The name of the limiter which needs to be applied.
```

Comment:
so to someone new to this area, (like me), its somewhat confusing that this talks about the name of the limiter, but its called PolicyName...  are limiters considered Policies?